fdbclient/SystemData.h

```diff
@@ -20,6 +20,7 @@
 
 #ifndef FDBCLIENT_SYSTEMDATA_H
 #define FDBCLIENT_SYSTEMDATA_H
+#include <_types/_uint16_t.h>
```

Comment:
Is this a bogus clangd insert? I've been passing `-header-insertion=never` to clangd to avoid this

Reply:
Yes, I need to figure out how to pass this argument to `clangd` in emacs... It's super annoying...